PWGJE/Tasks/PhiInJetsJE.cxx

```diff
@@ -0,0 +1,418 @@
+    JEhistos.add("hUSS_INSIDE_1D" ,"hUSS_INSIDE_1D", kTH1F, {MinvAxis});
+    JEhistos.add("hUSS_INSIDE_1D_2_3" ,"hUSS_INSIDE_1D_2_3", kTH1F, {MinvAxis});
+
+    JEhistos.add("hLSS_INSIDE" ,"hLSS_INSIDE", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    JEhistos.add("hLSS_INSIDE_1D" ,"hLSS_INSIDE_1D", kTH1F, {MinvAxis});
+    JEhistos.add("hLSS_INSIDE_1D_2_3" ,"hLSS_INSIDE_1D_2_3", kTH1F, {MinvAxis});
+
+    JEhistos.add("hUSS_OUTSIDE" ,"hUSS_OUTSIDE", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    JEhistos.add("hUSS_OUTSIDE_1D" ,"hUSS_OUTSIDE_1D", kTH1F, {MinvAxis});
+    JEhistos.add("hUSS_OUTSIDE_1D_2_3" ,"hUSS_OUTSIDE_1D_2_3", kTH1F, {MinvAxis});
+
+    JEhistos.add("hLSS_OUTSIDE" ,"hLSS_OUTSIDE", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    JEhistos.add("hLSS_OUTSIDE_1D" ,"hLSS_OUTSIDE_1D", kTH1F, {MinvAxis});
+    JEhistos.add("hLSS_OUTSIDE_1D_2_3" ,"hLSS_OUTSIDE_1D_2_3", kTH1F, {MinvAxis});
+
+    JEhistos.add("hMultFT0M", "hMultFT0M", kTH1F, {MultAxis});
+    
+
+
+    LFhistos.add("etaHistogram", "etaHistogram", kTH1F, {axisEta});
+    LFhistos.add("phiHistogram", "phiHistogram", kTH1F, {axisPhi});
+    LFhistos.add("nEvents", "nEvents", kTH1F, {{4, 0.0, 4.0}});
+    LFhistos.add("hDCArToPv", "DCArToPv", kTH1F, {{300,0.0,3.0}});
+    LFhistos.add("hDCAzToPv", "DCAzToPv", kTH1F, {{300,0.0,3.0}});
+    LFhistos.add("rawpT", "rawpT", kTH1F, {{1000,0.0,10.0}});
+    LFhistos.add("rawDpT", "rawDpT", kTH2F, {{1000,0.0,10.0}, {300,-1.5, 1.5}});
+    LFhistos.add("hIsPrim", "hIsPrim", kTH1F, {{2,-0.5,+1.5}});
+    LFhistos.add("hIsGood", "hIsGood", kTH1F, {{2,-0.5,+1.5}});
+    LFhistos.add("hIsPrimCont", "hIsPrimCont", kTH1F, {{2,-0.5,+1.5}});    
+    LFhistos.add("hFindableTPCClusters" ,"hFindableTPCClusters", kTH1F, {{200, 0, 200}});    
+    LFhistos.add("hFindableTPCRows" ,"hFindableTPCRows", kTH1F, {{200, 0, 200}});
+    LFhistos.add("hClustersVsRows" ,"hClustersVsRows", kTH1F, {{200, 0, 2}});
+    LFhistos.add("hTPCChi2" ,"hTPCChi2", kTH1F, {{200, 0, 100}});
+    LFhistos.add("hITSChi2" ,"hITSChi2", kTH1F, {{200, 0, 100}});
+    
+    LFhistos.add("hUSS" ,"hUSS", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    LFhistos.add("hUSS_1D" ,"hUSS_1D", kTH1F, {MinvAxis});
+    LFhistos.add("hUSS_1D_2_3" ,"hUSS_1D_2_3", kTH1F, {MinvAxis});
+
+    LFhistos.add("hLSS" ,"hLSS", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    LFhistos.add("hLSS_1D" ,"hLSS_1D", kTH1F, {MinvAxis});
+    LFhistos.add("hLSS_1D_2_3" ,"hLSS_1D_2_3", kTH1F, {MinvAxis});
+
+    LFhistos.add("hMultFT0M", "hMultFT0M", kTH1F, {MultAxis});
+    
+
+    
+    //EVENT SELECTION
+    eventSelection = JetDerivedDataUtilities::initialiseEventSelection(static_cast<std::string>(c_eventSelections));
+    // trackSelection = JetDerivedDataUtilities::initialiseTrackSelection(static_cast<std::string>(trackSelections));
+
+  
+  } //end of init
+
+  double massKa = TDatabasePDG::Instance()->GetParticle(kKPlus)->Mass();  // FIXME: Get from the common header
```

Comment:
Please fix this

Reply:
Dear Vit,

This has now been fixed.
Best Regards
Adrian.N